lib/tasks/check_license.rake

```diff
@@ -64,6 +64,9 @@ def license_report
     elsif fn =~ /\.md\z/ || fn =~ /\.doc\z/ || fn =~ /\.txt\z/
       report[:skipped] << "#{fn}: skipped by name match (documentation file)"
       next
+    elsif fn =~ /\.svg\z/ || fn =~ /\.eps\z/
+      report[:skipped] << "#{fn}: skipped by name match (vectorial image)"
```

Comment:
The correct term is `vector`. (e,g, https://en.wikipedia.org/wiki/Vector_graphics)
```suggestion
      report[:skipped] << "#{fn}: skipped by name match (vector image)"
```